packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -347,11 +347,32 @@ bool _checkXcodeVersion() {
   return true;
 }
 
+final String noCocoaPodsConsequence = '''
+  CocoaPods is used to retrieve the native platform side's plugin code that responds to your
+  plugin usage on the Dart side.
+  Without resolving native dependencies with CocoaPods, plugins usage will not work on iOS. 
```

Comment:
I'm not sure the word native is necessary here and that words is a bit overloaded.